src/libraries/System.Security.Cryptography.X509Certificates/tests/TestData.cs

```diff
@@ -7,6 +7,8 @@ namespace System.Security.Cryptography.X509Certificates.Tests
 {
     internal static class TestData
     {
+        private static readonly bool IsRC2Supported = !PlatformDetection.IsAndroid;
```

Comment:
Could we use the existing check we have in RC2Factory, or would it be impractical to do so?

Reply:
Maybe putting something like `IsRC2Supported` in `PlatformSupport.cs` and then making this and `RC2Factory` use that value?

Using RC2Factory directly seemed weird/awkward, since these tests would then need to define some IRC2Provider which they don't actually care about.

Reply:
I think putting it in PlatformSupport.cs would be a good idea.